strongbox-security/strongbox-security-api/src/test/java/org/carlspring/strongbox/security/certificates/KeyStoreManagerIntegrationTestIT.java

```diff
@@ -103,7 +102,7 @@ public void testWithoutProxy()
                                                             InetAddress.getLocalHost(),
                                                             LDAPS_PORT);
 
-        assertEquals(1, ks.size(), "localhost should have three certificates in the chain");
+        assertThat(ks.size()).as("localhost should have three certificates in the chain").isEqualTo(1);
```

Comment:
`assertThat(ks.size()).as("localhost should have three certificates in the chain").isEqualTo(1);`

=>

`assertThat(ks).as("localhost should have three certificates in the chain").hasSize(1);`

please change it wherever you use `.size()` on collections

Reply:
Changed assertions with `.size()` method.
In case of `ks.size()` it cannot be changed - KeyStore class doeas not implement Iterable